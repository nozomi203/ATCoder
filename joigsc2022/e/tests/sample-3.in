5 20
7
10
4
6
8
