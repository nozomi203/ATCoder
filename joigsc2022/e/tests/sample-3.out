2 18
2 8
1 18
1 14
1 8
